test/expected/partials_path_regex.js

```diff
@@ -1,14 +1,15 @@
 this["JST"] = this["JST"] || {};
 
 Handlebars.registerPartial("partial", Handlebars.template({"compiler":[6,">= 2.0.0-beta.1"],"main":function(depth0,helpers,partials,data) {
```

Comment:
+1 We definitely need to make the tests execute the compiled template and test the output of that, not test the body of the compiled template.

That said, it's been almost a month since Handlebars 3.0.0 came out, and I'd rather see this merged now and the tests fixed later than wait another week.